ghedesigner/ground_loads.py

```diff
@@ -305,7 +288,7 @@ def perform_current_month_simulation(
         delta_t_fluid_nom_max = max(delta_t_fluid_nom)
 
         if delta_t_fluid_nom_max > 0.0:
-            f = interp1d(delta_t_fluid_peak, hour_time)
+            f = interp1d(delta_t_fluid_peak, hour_time, fill_value="extrapolate")
```

Comment:
This is the "fix" in this PR, although this is more of a band-aid than a true solution.

In the lines of code above this, `q_peak` and `q_nominal` are equal out to about 10 decimal places, after which, the `q_nominal` remaining digits cause it to be greater than the `q_peak` value. Due to this, the subsequent calculations cause the interpolation to run out of range. Adding this allows the interpolation to run beyond the end of the range and not throw the error.

As I said, I think this is a temporary measure and not a true solution. The hybrid load scheme is soon to get a lot more scrutiny and potentially a full rewrite, so this code may not be around for long anyway.